src/mpid/ch4/netmod/ofi/util.c

```diff
@@ -242,17 +242,12 @@ int MPIDI_OFI_control_handler(void *am_hdr, void *data, MPI_Aint data_sz,
     }
 
     switch (ctrlsend->type) {
-        case MPIDI_OFI_CTRL_HUGEACK:{
-                /* FIXME: need vni from the callback parameters */
```

Comment:
Do we not need the VNI info anymore?

Reply:
Yes, we need it to reply to the right vni. Since this is native path, we can derive the vni info from the message envelope, or we can get it from the active message callback attr, or we can get it from the `remote_info`. We probably don't need to store the vni in the `remote_info`, but some redundancy is not bad.